crates/rpc/src/types/query.rs

```diff
@@ -1,6 +1,20 @@
+use alloy_primitives::B256;
 use serde::Deserialize;
 
 #[derive(Debug, Deserialize)]
 pub struct RandaoQuery {
     pub epoch: Option<u64>,
 }
+
+#[derive(Debug, Deserialize)]
+pub struct SlotQuery {
+    pub slot: Option<u64>,
+}
+#[derive(Debug, Deserialize)]
+pub struct RootQuery {
+    pub root: Option<B256>,
+}
+#[derive(Debug, Deserialize)]
+pub struct ParentRootQuery {
+    pub parent_root: Option<B256>,
+}
```

Comment:
```suggestion
#[derive(Debug, Deserialize)]
pub struct SlotQuery {
    pub slot: Option<u64>,
}

#[derive(Debug, Deserialize)]
pub struct RootQuery {
    pub root: Option<B256>,
}

#[derive(Debug, Deserialize)]
pub struct ParentRootQuery {
    pub parent_root: Option<B256>,
}
```

I shouldn't need to comment on this as often as I do